### PR TITLE
Introduce viaq_index_name field

### PR DESCRIPTION
**Pull request by lukas-vlcek**

### CODE REVIEW

**Comment by lukas-vlcek** [comment]
FYI, @mijicherian I fixed the tests for #115

File: namespaces/_default_.yml [hunk 1]
```diff
@@ -187,3 +187,10 @@ _index_type_:
         store or application other than Elasticsearch, but you still need
         to correlate data with the data stored in Elasticsearch, this field
         will give you the exact document corresponding to the record.
+        
+    - name: viaq_index_name
+      type: keyword
+      example: project.my-cool-project-in-lab04.748e92c2-70d7-11e9-b387-000d3af2d83b.2019.05.09
+      description: |
+        Index name in which this message will be stored within the elastic search.
```
Comment (richm) [inline namespaces/_default_.yml]: elasticsearch

Comment (lukas-vlcek) [reply namespaces/_default_.yml]: Fixed

**Comment by mijicherian** [comment]
Thanks @lukas-vlcek for your efforts.

**Comment by lukas-vlcek** [comment]
@richm PTAL

**Review by richm — APPROVED** [review]
/lgtm

**Comment by lukas-vlcek** [comment]
@richm  @mijicherian  When we are going to need this new field? (i.e. when we need to cut a new model release?)

**Comment by richm** [comment]
> @richm @mijicherian When we are going to need this new field? (i.e. when we need to cut a new model release?)

I would say now - at least, go ahead and generate a release, and submit a PR to origin-aggregated-logging to use the new release

File: namespaces/_default_.yml [hunk 2]
```diff
@@ -187,3 +187,10 @@ _index_type_:
         store or application other than Elasticsearch, but you still need
         to correlate data with the data stored in Elasticsearch, this field
         will give you the exact document corresponding to the record.
+        
+    - name: viaq_index_name
+      type: keyword
+      example: project.my-cool-project-in-lab04.748e92c2-70d7-11e9-b387-000d3af2d83b.2019.05.09
+      description: |
+        Index name in which this message will be stored within the Elasticsearch.
```
Comment (lukas-vlcek) [inline namespaces/_default_.yml:195]: @richm @jcantrill How is this going to be impacted when moving to the new data model with ES 6.x (rollover and common index; no more index per namespace/day).

Comment (richm) [reply namespaces/_default_.yml:195]: we still need this field

Comment (jcantrill) [reply namespaces/_default_.yml:195]: https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/cluster-logging-es-rollover-data-design.md#data-model

**Pull request merged**